pyanaconda/ui/gui/spokes/lib/cart.py

```diff
@@ -217,3 +235,26 @@ def on_set_as_boot_clicked(self, button):
             self._previousID = self._store[itr][ID_COL]
 
         self._toggle_button_text(self._store[itr])
+
+    def _initialize_zipl_secure_boot(self):
+        if not arch.is_s390():
+            self._secure_boot_box.hide()
+            return
+
+        secure_boot = self._bootloader_proxy.ZIPLSecureBoot
+
+        if secure_boot == SECURE_BOOT_DEFAULT:
+            secure_boot = SECURE_BOOT_AUTO
```

Comment:
Not sure if the GUI should be responsible about what means `DEFAULT`. Shouldn't it be more like the `DEFAULT` is decided by the module and module will give you the "real" value?

Reply:
Fixed.